patrol_node:
  ros__parameters:
    initial_point: [0.0, 0.0, 0.0]
    target_points: [
      0.0,0.0,0.0,
      1.0,1.0,1.57,
      -4.0,-5.0,1.0,
      -13.0,-5.0,3.0,
      -8.0,-10.0,0.5,
      0.0,0.0,0.0,
      ]